osutil/mkfs/mkfs_test.go

```diff
@@ -103,6 +104,23 @@ func (m *mkfsSuite) TestMkfsExt4Happy(c *C) {
 		},
 	})
 
+	cmd.ForgetCalls()
```

Comment:
These kinds of cases must be their own unit test, and not a part of a Happy case

Reply:
OK. The definition of the "Happy case" here is a bit blurry to me. I am used to consider "happy paths" as the ones not leading to errors but here I understand this is more like the "straight" path.